apps/docs/src/resources/icons.md

```diff
@@ -0,0 +1,39 @@
+# Icons
+
+onyx includes a set of well-crafted SVG icons that are free to use.
+
+If you want to download the icons as SVG, please head over to our [GitHub repository](https://github.com/SchwarzIT/onyx/tree/main/packages/icons/src/assets).
+
+A full changelog can be found [here](/development/packages/changelogs/icons).
+
+## Installation
+
+To get started, install the npm package with your corresponding package manager:
+
+::: code-group
+
+```sh [pnpm]
+pnpm add -D @sit-onyx/icons@alpha
+```
+
+```sh [npm]
+npm install -D @sit-onyx/icons@alpha
+```
+
+```sh [yarn]
+yarn install -D @sit-onyx/icons@alpha
+```
+
+:::
+
+Afterwards, you can import and use icons as needed. Please see the [OnyxIcon](/development/OnyxIcon) for the technical documentation.
+
+## Available icons
+
+You can hover over an icon to see its name.
```

Comment:
hover is not implemented. remove this text then?

Reply:
If you hover over it the `title` attribute will be shown :) Thats the browser default

Reply:
afaik the hover behaviour is OS dependent

Reply:
Maybe @jannick-ux has a nice idea how to show the icon names on hover. Maybe similar to the design tokens? 
Could be implemented together with https://github.com/orgs/SchwarzIT/projects/5/views/7?layout=board&pane=issue&itemId=51923734

Reply:
ok, I added this info to the ticket. it's out of scope of this PR then :)

Reply:
Yes I will create a concept for promoting the icons